app/workers/reject_awaiting_references_course_choices_worker.rb

```diff
@@ -0,0 +1,9 @@
+class RejectAwaitingReferencesCourseChoicesWorker
+  include Sidekiq::Worker
+
+  def self.perform
```

Comment:
```suggestion
  def perform
```
This fixes the error in the Sidekiq worker for me locally